setup.py

```diff
@@ -50,7 +50,7 @@ def get_version(*file_paths):
     include_package_data=True,
     install_requires=[
         # TBD: GH issue #3 includes support for elasticsearch-dsl>=6.2.0
-        "Django>=1.8,<=2.2", "elasticsearch-dsl>=6.0.0,<6.2.0", "texttable>=1.2.1",
+        "Django>=1.8,<=2.2.19", "elasticsearch-dsl>=6.0.0,<6.2.0", "texttable>=1.2.1",
```

Comment:
Consider opening it up a little more, I highly doubt anything in the package is incompatible with Django 3.0 / 3.1